### PR TITLE
Adding accessibility standard - Keyboard navigation

### PR DESCRIPTION
**Pull request by sanat-thukral**
# Adding keyboard navigation for accessibility

#134 

## Description

Adding keyboard navigation via tab buttons and arrow keys to improve accessibility

## Checklist for Pull Requests

- [x] Supplied as many details as possible on this change
- [x] Included the link to the associated issue in the Issue section above
- [x] The code conforms to the general [development guidelines](https://github.com/SAP/guided-answers-extension/blob/main/docs/developer-guide.md).
- [ ] The code has **unit tests** where applicable and is easily unit-testable
- [x] This branch is appropriately named for the associated issue

### CODE REVIEW

File: packages/webapp/src/webview/ui/components/Header/NavigationButtons/NavigationButtons.tsx [hunk 1]
```diff
@@ -7,7 +7,15 @@ import i18next from 'i18next';
  *
  * @returns A button component navigating to all answers.
  */
-export function AllAnswersButton() {
+const AllAnswersButton = React.memo(function AllAnswersButton() {
+    requestAnimationFrame(() => {
+        //(document.body.querySelector('button') as HTMLElement).focus();
```
Comment (unseen1980) [inline packages/webapp/src/webview/ui/components/Header/NavigationButtons/NavigationButtons.tsx:12]: Maybe remove this?

Comment (sanat-thukral) [reply packages/webapp/src/webview/ui/components/Header/NavigationButtons/NavigationButtons.tsx]: Thanks for pointing that out. The comment has now been removed

**Comment by unseen1980** [comment]
I pulled the branch locally and everything works as expected. Code is looking good. Some missing tests around focus can be added later. We need a good way to mock document object. 

Thanks @sanat-thukral , great work 👍

**Review by unseen1980 — APPROVED** [review]

**Comment by sanat-thukral** [comment]
Thanks for the review Christos 😄

**Pull request merged**